Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -691,6 +691,8 @@ def write(self, writer, font, tableDict, value, repeatIndex=None):
         assert writer.items[lengthIndex] == b"\xde\xad\xbe\xef"[: conv.staticSize]
         writer.items[lengthIndex] = lengthWriter.getAllData()
 
+    def __repr__(self):
```

Comment:
I recomment to make a separate PR for changes other than type annotations.

Reply:
You are right. I tried to avoid changes like this …